SMLHelper/Patchers/TechDataPatcher.cs

```diff
@@ -0,0 +1,74 @@
+﻿namespace SMLHelper.V2.Patchers
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+    using Assets;
+    using Harmony;
+
+
+    internal class TechDataPatcher
```

Comment:
**TODO** Wrap this entire file in 
```CS
#if BELOWZERO
// code here
#endif
```

Reply:
This file was removed - All of its patching was moved into CraftDataPatcher to keep related patches in the same space